packages/sanity/src/desk/panes/document/useDocumentTitle.ts

```diff
@@ -13,33 +13,34 @@ interface UseDocumentTitle {
 }
 
 /**
- * React hook that returns the document title for the current document in the document pane.
+ * React hook that returns the _preview_ title for the current document in the document pane
+ * based off the current `formState` value.
  *
  * @beta
  * @hidden
  *
  * @returns The document title or error. See {@link UseDocumentTitle}
  */
 export function useDocumentTitle(): UseDocumentTitle {
-  const {connectionState, schemaType, title, value: documentValue} = useDocumentPane()
-  const subscribed = Boolean(documentValue) && connectionState === 'connected'
+  const {connectionState, formState, schemaType} = useDocumentPane()
+  const formStateValue = formState?.value
+
+  const subscribed = Boolean(formStateValue) && connectionState === 'connected'
 
   const {error, value} = useValuePreview({
     enabled: subscribed,
     schemaType,
-    value: documentValue,
+    value: formStateValue,
   })
 
-  if (connectionState !== 'connected') {
-    return {error: undefined, title: undefined}
-  }
-
-  if (title) {
-    return {error: undefined, title}
-  }
-
-  if (!documentValue) {
-    return {error: undefined, title: `New ${schemaType?.title || schemaType?.name}`}
+  // Don't return a title if unable to retrieve the current preview
+  // or if the `formState` value doesn't contain `_updatedAt` - indicating
+  // that the current `formState` represents a newly created document.
+  if (connectionState !== 'connected' || !formStateValue?._updatedAt) {
```

Comment:
Here, we're checking for the existence of `updatedAt` to determine if the current form state represents a brand new document or not. 

I considered using the `timelineStore` to determine this (by grabbing the current `revTime` and checking if it's of an `initial` type), but it felt a little overkill. Perhaps there's a better / more idiomatic way to do this?